.gitattributes

```diff
@@ -0,0 +1 @@
+*.bin binary
```

Comment:
Don't need necessarily, but git diff was showing me compiled protodescriptors as pain text. Note that this is `.gitattributes`, not `.gitignore`: it doesn't ignore anything, it just says "binary files differ" on files that end in .bin instead of attempting to show you text diff on binary files, which I think is a nice to have here.

Reply:
Ahh makes sense -- feel free to add that if it helps then.